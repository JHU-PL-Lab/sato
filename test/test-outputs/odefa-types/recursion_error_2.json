[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "tru" ],
          "err_match_val": "true",
          "err_match_expected": "int",
          "err_match_actual": "bool"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "v = a - tru",
    "err_steps": 560
  }
]