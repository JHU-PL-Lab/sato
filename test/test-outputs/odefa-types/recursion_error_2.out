Type errors for:
- Input sequence  : []
- Found at clause : v = a - tru
--------------------
* Value    : tru = true
* Expected : int
* Actual   : bool
1 errors found. No further control flows exist.
